torch/csrc/jit/serialization/pickler.h

```diff
@@ -296,8 +297,17 @@ uint64_t getStorageKey(const at::Tensor& tensor);
 // otherwise return false
 bool checkHasValidSetGetState(const std::shared_ptr<c10::ClassType>& cls);
 
-// Return a map of Tensor Metadata for serialization.
-// For now, it only takes care of `conj` and `neg` bit.
+// Declare BackendMate serialization and deserialization function pointer types.
```

Comment:
nit: BackendMeta

Reply:
Thanks for the correction, I have fixed such typos.